docs/source/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -190,8 +190,8 @@ The output PDF can be customized with the following parameters:
 - By calling with `merge_banks=True` a PDF will be generated based on the weighted sum of the detector
   banks performed using supplied Q limits `q_lims=q_limits`, q_limits can be in the form of an array or
   with shape (2, x) where x is the number of detectors, or a string containing the directory
```

Comment:
I think this should say where x is the number of banks rather than detectors? Not your piece of text but noticed it because you've repeated "detectors" in the extra text you've added. Please can you change both to banks